### ISSUE TITLE
"make deploy" fails with "invalid patches from <nil>"

### ISSUE THREAD

**Issue by julio-santiesteban** [xridge/kubestone#28]
Bug when trying to deploy with kustomize:
```
kustomize build config/default | kubectl apply -f -
2019/08/05 10:30:16 invalid patches from <nil>
error: no objects passed to apply
make: *** [Makefile:37: deploy] Error 1
```

**Comment by otto-von-bivouac** [comment]
This is caused by too new kustomize version in some occasion.

Could you please make a try with kustomize 2.1.0?

If that resolves, we should include this limitation in #18

**Comment by julio-santiesteban** [comment]
@otto-von-bivouac yes, it works with kustomize 2.1.0. Thanks!
